Documentation/MigrationGuide.md

```diff
@@ -1,15 +1,116 @@
-# Master
+# SwiftGen 6.0 Migration Guide
+
+If you're migrating from SwiftGen 5.x to SwiftGen 6.0, there might be some migration steps you'll need to use.
+
+Below is a list of pointers to help you migrate to the new SwiftGen 6.0
+
+## `storyboards` command has been renamed
+
+The `storyboards` subcommand has been renamed `ib` (for Interface Builder).
```

Comment:
The new command `ib` is able to parse `.xib` files in addition to `.storyboard` files.

Reply:
It's not able to (yet), though.